The V4L2 tuner norms that a device supports could be a subset of some norm (e.g. NTSC instead of NTSC_M).The comparison should be done by & instead of ==.

See: http://bugzilla.gnome.org/show_bug.cgi?id=569820

diff -uNr orig/gst-plugins-good-0.10.10/sys/v4l2/gstv4l2tuner.c gst-plugins-good-0.10.10/sys/v4l2/gstv4l2tuner.c
--- orig/gst-plugins-good-0.10.10/sys/v4l2/gstv4l2tuner.c	2007-07-18 06:42:33.000000000 -0500
+++ gst-plugins-good-0.10.10/sys/v4l2/gstv4l2tuner.c	2009-01-29 16:11:21.000000000 -0600
@@ -244,7 +244,7 @@
   gst_v4l2_get_norm (v4l2object, &norm);
 
   for (item = v4l2object->norms; item != NULL; item = item->next) {
-    if (norm == GST_V4L2_TUNER_NORM (item->data)->index)
+    if (norm & GST_V4L2_TUNER_NORM (item->data)->index)
       return (GstTunerNorm *) item->data;
   }
 
